### DISCUSSION TITLE
Nerf pp for Autopilot similar to Relax

### DISCUSSION THREAD

**Discussion by hlysine** [ppy/osu#16509] [Ideas] [answered]
I noticed this when working on #16483 

![osu! 18_1_2022 11_08_43 pm](https://user-images.githubusercontent.com/25472513/149963504-ea8af1c9-2f2d-451b-a22a-7127dae8da6b.png)

Currently, enabling autopilot mod has no effect on pp, while enabling relax mod causes the accuracy performance to be 0, with an additional 0.6x multiplier applied to total pp. Autopilot should get the same treatment, e.g. setting aim performance to 0 and adding a multiplier to total pp, since it doesn't make sense for autopilot to grant aim performance.

**Comment by apollo-dw** [comment, marked as answer]
This is planned - I actually have a branch for this but I haven't pushed it yet... can do soon
